.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
```

Comment:
:thinking: What do you think about this?
```suggestion
Closes <ticket>
```

Reply:
I like it!